conf/application.conf

```diff
@@ -146,8 +146,8 @@ play.i18n.langs = ["en", "cy"]
 features.welsh-language-support = true
 
 timeout {
-  period = 900
-  countDown = 120
+  period = 30
+  countDown = 10
 }
```

Comment:
So do we not change timeout value in penalties-frontend service as well in [https://github.com/hmrc/penalties-frontend/blob/1a05d76cb9abdb631a03d46fa36b3da6cc2892fb/conf/application.conf#L146](url)